front/app/modules/commercial/content_builder/admin/containers/index.tsx

```diff
@@ -14,49 +14,57 @@ import ContentBuilderToolbox from '../components/ContentBuilderToolbox';
 import ContentBuilderSettings from '../components/ContentBuilderSettings';
 import Container from '../components/CraftComponents/Container';
 import RenderNode from '../components/RenderNode';
+import ContentBuilderTopBar from '../components/ContentBuilderTopBar';
 
 const Wrapper = styled.div`
   flex: 0 0 auto;
-  width: 210px;
+  width: 212px;
 `;
 
 const ContainerInner = styled.nav`
   flex: 0 0 auto;
-  width: 210px;
+  width: 212px;
   display: flex;
   flex-direction: column;
   align-items: center;
   position: fixed;
   top: 0;
   bottom: 0;
   padding-top: ${stylingConsts.menuHeight + 10}px;
-  background-color: ${colors.disabledPrimaryButtonBg};
-  border-right: 1px solid ${colors.border};
+  background-color: ${colors.separation};
+  border-right: 1px solid ${colors.separation};
 `;
 
 const ContentBuilderPage = () => {
   return (
-    <Editor resolver={{ Box, Container }} onRender={RenderNode}>
-      <Wrapper>
-        <ContainerInner>
-          <ContentBuilderToolbox />
-        </ContainerInner>
-      </Wrapper>
-      <RightColumn>
-        <Frame>
-          <Element
-            is="div"
-            canvas
-            style={{
-              padding: '4px',
-              minHeight: '300px',
-              backgroundColor: '#fff',
-            }}
-          />
-        </Frame>
-      </RightColumn>
-      <ContentBuilderSettings />
-    </Editor>
+    <>
+      <Box position="absolute" zIndex="1" w="100%">
```

Comment:
The problem with this is that it produces weird scrolling behaviour once the page becomes scrollable :)